test/widgets/content_test.dart

```diff
@@ -158,6 +166,78 @@ void main() {
     });
   });
 
+  group('Internal links', () {
```

Comment:
The other groups in this test file are named for the corresponding classes for the pieces of content they test. This should follow that, to help keep the file organized.

E.g. it could be "LinkNode on internal links", if it's not combined into the existing "LinkNode interactions".